config.yaml

```diff
@@ -0,0 +1,10 @@
+buildConfiguration:
+  buildCommand: cd src/components/puck/registry && npm run generate
```

Comment:
Let's continue to use pnpm instead of npm.

Reply:
Is there a way to have the build step in the platform use pnpm?

Reply:
https://yext.slack.com/archives/CC7B2751A/p1731077760800839

Reply:
switched